presto-product-tests/src/main/resources/sql-tests/datasets/table_with_header.data-revision

```diff
@@ -0,0 +1 @@
+v.1.0
```

Comment:
what's that?

Reply:
tempto... if data revision did not change then it does not update data on hdfs when populating test tables. Optimization.